src/lib/db/client-applications-store.ts

```diff
@@ -412,7 +412,7 @@ export default class ClientApplicationsStore
             return acc;
         }, []);
         environments.forEach((env) => {
-            env.sdks?.sort();
+            env.sdks.sort();
```

Comment:
According to `Server failed executing request: {} Cannot read properties of null (reading 'sort')","name":"controller/MetricsController","plan":"Enterprise","region":"eu-central-1","stack":"TypeError: Cannot read properties of null (reading 'sort')\n    at /unleash/node_modules/unleash-server/dist/lib/db/client-applications-store.js:330:22\n    at Array.forEach (<anonymous>)\n    at ClientApplicationsStore.mapApplicationOverviewData (/unleash/node_modules/unleash-server/dist/lib/db/client-applications-store.js:329:22)\n    at ClientApplicationsStore.getApplicationOverview (/unleash/node_modules/unleash-server/dist/lib/db/client-applications-store.js:291:21)` there seems to be some edge cases where this is still undefined